workspaces/adventure-pack/goodies/java/src/digits_int/Test.java

```diff
@@ -0,0 +1,72 @@
+package digits_int;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+
+class DigitsIntTest {
+
+  @Test
+  public void withIntegerGreaterThanZero() {
+    try {
+      var digits = AP.digits(4984);
```

Comment:
If we add `import static digits_int.AP.digits;` among the imports, I believe we'll be able to simply do `digits(4984)` here.

```suggestion
      var digits = digits(4984);
```

Reply:
Addressed.